ajs-app.ts

```diff
@@ -1,40 +1,43 @@
+/* eslint-disable angular/no-controller */
```

Comment:
Do not disable the eslint rules, unless you really want to mark an exception from the rule. Even though you are not going to fix the errors and warnings right now, just let them shine, so we know that there is some code that should be changed eventually. Thanks.

Reply:
Got it! I thought that wasn't a big deal, but i will keep that in mind!